modules/querying/pages/func/type-conversion-functions.adoc

```diff
@@ -189,4 +189,116 @@ Converts a number to a string using the C++ printf function with the `g` specifi
 to_string(1000.1) -> "1000.1"
 to_string(0.0000000001) -> "1E-10"
 to_string(12300000000) -> "1.23E7"
-----
\ No newline at end of file
+----
+
+== `toBoolean()`
+
+  
+===  Syntax
+
+`toBoolean(str)`
+
+  
+===  Description
+
+This function converts a string into a boolean value (`true` or `false`).
+
+* If the input is already a boolean, it returns the same value.
+* If the input is a string, the function tries to parse it into a boolean. For example, "true" becomes true, and "false" becomes false.
+
+  
+===  Return type
+
+`BOOLEAN`
+  
+===  Parameters
+
+|===
+| Parameter | Description | Data type
+
+| `str`
+| The string to convert into a boolean
+| String
+|===
+
+=== Example
+
+----
+toBoolean("true") -> true  
+toBoolean("false") -> false  
+toBoolean("invalid") -> null
+----
+
+== `toFloat()`
+
+  
+===  Syntax
+
+`toFloat(num)`
+
+  
+===  Description
+
+Converts an integer or string to a floating-point number.
+
+* If the input is already a floating-point number, the function will return it unchanged.
+* If the input is a valid number in string format (e.g., "14.5"), the function converts it to a float (e.g., 14.5).
+
+  
+===  Return type
+
+`FLOAT`
+  
+===  Parameters
+
+|===
+| Parameter | Description | Data type
+
+| `num`
+| The number to convert into a float
+| Number
+|===
+
+=== Example
+
+----
+toFloat(14) -> 14.0  
+toFloat("-3.5") -> -3.5  
+----
+
+== `toInteger()`
+
+  
+===  Syntax
+
+`toInteger(num)`
+
+  
+===  Description
+
+Converts a floating-point number or string to an integer.
+
+* If the input is already an integer, it is returned unchanged.
+* If the input is a valid number in string format (e.g., "42"), the function converts it to an integer (e.g., 42).
+
+  
+===  Return type
+
+`INTEGER`
+  
+===  Parameters
+
+|===
+| Parameter | Description | Data type
+
+| `num`
+| The number to convert into an integer
+| Number
+|===
+
+=== Example
+
+----
+toInteger(14.7) -> 14  
+toInteger("-10") -> -10
```

Comment:
```suggestion
toInteger("-10") -> -10
toInteger("five") -> NULL
```
It's good to include an example of an input that a user might think is valid, but isn't.